### PR TITLE
Model validation added

### PR DESCRIPTION
**Pull request by sabyanna**

### CODE REVIEW

File: ferrilata-devilline/Controllers/BadgesController.cs [hunk 1]
```diff
@@ -1,4 +1,6 @@
 ﻿using ferrilata_devilline.Services.Interfaces;
+using Microsoft.AspNetCore.Authentication.Google;
```
Comment (kskulikova) [inline ferrilata-devilline/Controllers/BadgesController.cs]: Unused authentication-related imports left (line 2-3)

File: ferrilata-devilline/Controllers/BadgesController.cs [hunk 2]
```diff
@@ -14,7 +16,7 @@ public BadgesController(IBadgeService badgeService)
 
         [HttpGet]
         [Route("/api/badges")]
-        public IActionResult getGadgets()
+        public IActionResult getBadges()
```
Comment (kskulikova) [inline ferrilata-devilline/Controllers/BadgesController.cs]: Styleguide (**G**etBadges vs **g**etBadges)

File: ferrilata-devilline/Controllers/PitchController.cs [hunk 3]
```diff
@@ -24,9 +24,9 @@ public IActionResult PostPitch([FromBody] AuxPitch NewPitch)
                 return Unauthorized(new {message = "Unauthorized"});
             }
 
-            if (HelperMethods.HelperMethods.checkMissingPostedPitchFields(NewPitch))
+            if (!ModelState.IsValid)
             {
-                return NotFound(new {error = "Please provide all fields"});
+                return NotFound(new { error = "Please provide all fields" });
```
Comment (kskulikova) [inline ferrilata-devilline/Controllers/PitchController.cs]: Extra spaces (line 29 inside the brackets)

File: ferrilata-devilline/Controllers/ApiAdminController.cs [hunk 4]
```diff
@@ -1,29 +1,31 @@
...
 
-                return Created("/api/badges/1", new List<object> { new { message = "Created" } });
+            if (!ModelState.IsValid)
+            {
+                return BadRequest(new { error = "Please provide all fields" });
             }
 
-            return Unauthorized(Json(new { error = "Unauthorized" }));
+            return Created("/api/badges/1", new List<object> { new { message = "Created" } });
         }
```
Comment (kskulikova) [inline ferrilata-devilline/Controllers/ApiAdminController.cs]: Styleguide: extra spaces inside the curly braces (lines 28, 25, 20)

File: ferrilata-devilline/Startup.cs [hunk 5]
```diff
@@ -101,6 +101,7 @@ public void ConfigureTestingServices(IServiceCollection services)
 
             services.AddScoped<IBadgeService, MockBadgeService>();
             services.AddScoped<IPitchService, MockPitchService>();
+
```
Comment (kskulikova) [inline ferrilata-devilline/Startup.cs:111]: Styleguide  - extra empty line

**Review by kskulikova — APPROVED** [review]

**Review by AlexTeper — APPROVED** [review]

**Comment by AlexTeper** [comment]
@sabyanna please resolve the merge conflicts

**Pull request merged**